### DISCUSSION TITLE
Next release date ?

### DISCUSSION THREAD

**Discussion by reynadan** [foundation/foundation-emails#1082] [General]
Hello guys,

Just a simple question, do you know when will be the next release maybe ?
There are some updates i'm interested in

Cheers

**Comment by joeworkman** [comment]
I am shooting for a January update.

**Reply by stinhambo** [reply]
How's the update coming along Joe?

Any particular fixes we should look forward to such as Gmail responsiveness?

**Reply by joeworkman** [reply]
I am starting work on this today.

**Reply by stinhambo** [reply]
Excellent! I'm starting a project which requires Foundation for Email but I was reluctant due to lack of active development.
